src/App.js

```diff
@@ -1,88 +1,86 @@
-import React from "react"
-import Die from "./Die"
-import {nanoid} from "nanoid"
-import Confetti from "react-confetti"
+import React from "react";
+import Die from "./Die";
+import { nanoid } from "nanoid";
+import Confetti from "react-confetti";
 
 export default function App() {
+  const [dice, setDice] = React.useState(allNewDice());
```

Comment:
God Job!  But I think its better if you import useState from react instead of using React.useState(), also it applies in all hooks.